foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -87,7 +107,24 @@ MobileCore.updateConfiguration(data);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+#### Swift
+
+```swift
+let updatedConfig = ["global.privacy":"optedout"]
+MobileCore.updateConfiguration(configDict: updatedConfig)
```

Comment:
https://github.com/adobe/aepsdk-core-ios/blob/14dbeef70cb1375ea86667db5571e74522ccc23f/AEPCore/Sources/core/MobileCore%2BConfiguration.swift#L45
```swift
static func updateConfigurationWith(configDict: [String: Any])
```
The function name is `updateConfigurationWith` in Swift.